shop-catalog/frontends/react-redux/src/components/pagination/pagination.tsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
```

Comment:
Test `pagination` If it can work as standalone component and what are the bold the requirements.